### ISSUE TITLE
Suppress Handle Delete Dialogue

### ISSUE THREAD

**Issue by MrPrimate** [ElfFriend-DnD/foundryvtt-items-with-spells-5e#3]
Currently in the process of integrating the module into DDB Importer and I would like the ability to suppress the file deletion dialogue that pops up. When reimporting a character it clears down a lot of spells/data and would be good to be able to suppress these as the user won't care.

**Comment by akrigline** [comment]
Oh good call, while you're here, would anything else make your life easier for being able to import items "with spells" configured?

**Comment by MrPrimate** [comment]
Everything else pretty much works right now I think!

**Comment by akrigline** [comment]
Implemented the following in 1.1.0 (also noted in the readme):

## API
When programatically deleting items that have spells attached, it is possible to skip the prompt for the user by providing the following object in the [DocumentModificationContext](https://foundryvtt.com/api/classes/foundry.abstract.Document.html#delete) option of the `delete` method.

```
someItem.delete({
  itemsWithSpells5e: { alsoDeleteChildSpells: true } // or false
})
```

The logic within this module will respect the boolean provided, deleting the child item spells itself if `true`, or leaving them alone if `false`.

**Comment by MrPrimate** [comment]
Thanks, works great!
